### PR TITLE
Backports 0.18 pr14

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
A few relatively simple PRs

### CODE REVIEW

**Comment by UdjinM6** [comment]
Hmm... it looks like you skipped changes to `.travis*` files in 14252, 14673 and 14674. Why?

**Comment by PastaPastaPasta** [comment]
> Hmm... it looks like you skipped changes to `.travis*` files in 14252, 14673 and 14674. Why?

Normally I go on strike with regards to travis due to the substancial differences, and based on my IDE, the differences looked bad. When looking manually the changes are easy. Pushed / fped

File: .travis.yml [hunk 1]
```diff
@@ -278,7 +270,7 @@ after_success:
         RUN_BENCH=true
         RUN_FUNCTIONAL_TESTS=false # Disabled for now, can be combined with the other x86_64 linux NO_DEPENDS job when functional tests pass the sanitizers
```
Comment (UdjinM6) [inline .travis.yml:272]: 14252: original PR dropped this line too
```suggestion
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Comment by UdjinM6** [comment]
Looks good 👍 pls squash the fix

**Comment by PastaPastaPasta** [comment]
Done

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**